terraform/modules/rds/main.tf

```diff
@@ -0,0 +1,56 @@
+# Non-compliant RDS configuration (missing encryption, backups, and monitoring)
+resource "aws_db_instance" "main" {
+  identifier        = "${var.environment}-db"
+  engine           = "postgres"
+  engine_version   = "13.7"
+  instance_class   = var.instance_class
+  allocated_storage = 20
+
+  # Non-compliant: Credentials in plain text
```

Comment:
The RDS instance configuration includes hardcoded credentials, which is a security risk. Use AWS Secrets Manager or SSM Parameter Store to manage sensitive information securely.